autogpt_platform/backend/schema.prisma

```diff
@@ -601,26 +607,33 @@ model StoreListing {
   createdAt DateTime @default(now())
   updatedAt DateTime @default(now()) @updatedAt
 
-  isDeleted  Boolean @default(false)
-  // Not needed but makes lookups faster
-  isApproved Boolean @default(false)
+  isDeleted          Boolean @default(false)
+  // Whether any version has been approved and is available for display
+  hasApprovedVersion Boolean @default(false)
+  
+  // URL-friendly identifier for this agent (moved from StoreListingVersion)
+  slug        String  @unique
```

Comment:
Adding you another headache here: the slug was not uniquely guaranteed in StoreListingVersion, so it's likely this will fail on the db with pre-existing data.

Reply:
Bruh

Reply:
Is this fine as not unique because user account in url?